Source/WebGPU/WebGPU/Device.mm

```diff
@@ -163,12 +163,13 @@ static void captureFrame(id<MTLDevice> captureObject)
         RefPtr<Device> protectedThis = weakThis.get();
         if (!protectedThis)
             return;
-        auto& instance = protectedThis->instance();
-        instance.scheduleWork([protectedThis = WTFMove(protectedThis), device = device]() {
-            if (![protectedThis->m_device isEqual:device])
-                return;
-            protectedThis->loseTheDevice(WGPUDeviceLostReason_Undefined);
-        });
+        if (auto instance = protectedThis->instance(); instance.get()) {
```

Comment:
In the other examples you appear to set the precedent of `if (auto inst = ...; inst.get())`

Reply:
oh yes, I didn't like the abbreviation but it was to avoid the compiler error:
```
Variable 'instance' declared with deduced type 'auto' cannot appear in its own initializer
```
and I couldn't think of a better name. We avoid the abbreviation here because `this` is not in scope, but rather `protectedThis`